### PR TITLE
Add 2021-07-01 meeting notes (closes #269)

### LINKED ISSUE #269: TSC meeting 01-July-2021
# Time

UTC Thu 01-Jul-2021 20:00:
- Los Angeles: Thu 01-Jul-2021 13:00
- Chicago: Thu 01-Jul-2021 15:00
- New York: Thu 01-Jul-2021 16:00
- Madrid: Thu 01-Jul-2021 22:00
- Moscow: Thu 01-Jul-2021 23:00
- Tokyo: Fri 02-Jul-2021 05:00
- Sydney: Fri 02-Jul-2021 06:00

# Location

https://eslint.org/chat/tsc-meetings

# Agenda

Extracted from:

* Issues and pull requests from the ESLint organization with the ["tsc agenda" label](https://github.com/issues?utf8=%E2%9C%93&q=org%3Aeslint+label%3A%22tsc+agenda%22)
* Comments on this issue

# Invited

- Nicholas C. Zakas (@nzakas) - TSC
- Brandon Mills (@btmills) - TSC
- Milos Djermanovic (@mdjermanovic) - TSC

# Public participation

Anyone is welcome to attend the meeting as observers. We ask that you refrain from interrupting the meeting once it begins and only participate if invited to do so.

### PR DESCRIPTION
**Pull request by btmills**

### CODE REVIEW

File: notes/2021/2021-07-01.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+#### [--report-unused-disable-directives should be autofixable](https://github.com/eslint/eslint/issues/11815)
+
+* The [PR](https://github.com/eslint/eslint/pull/14617) was recently updated, and @mdjermanovic will take a look.
+
+#### [Allow all directives in line comments](https://github.com/eslint/eslint/issues/14575)
+
+* The [PR](https://github.com/eslint/eslint/pull/14656) just needs a small documentation update.
+* @nzakas will take a look soon.
+
+#### [Upgrade to Acv version 8](https://github.com/eslint/eslint/issues/13888)
```
Comment (aladdin-add) [inline notes/2021/2021-07-01.md:47]:
```suggestion
#### [Upgrade to Ajv version 8](https://github.com/eslint/eslint/issues/13888)
```

File: notes/2021/2021-07-01.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+### Contributor pool
+
+* @nzakas nominates @snitin315. Committer pay will kick in next month in July. :+1: from @btmills and @mdjermanovic.
+* @nzakas nominates @bmish. :+1: from @btmills and @mdjermanovic.
+* @nzakas nominates @jacknapis for Discord help. :+1: from @btmills and @mdjermanovic.
+* @mdjermanovic nominates @brettz9 for 3 ESM PRs. :+1: from @btmills and @nzakas.
+* @mdjermanovic nominates @ota-meshi for work to support the [regex d flag](https://github.com/eslint/eslint/pull/14653). :+1: from @btmills and @nzakas.
+* @mdjermanovic nominates @JoshuaKGoldberg for [fixable disable directives](https://github.com/eslint/eslint/pull/14617). :+1: from @btmills and @nzakas.
+* @mdjermanovic nominates @A-Katopodis for [exit on fatal error](https://github.com/eslint/eslint/pull/14730) :+1: from @btmills and @nzakas.
+* @mdjermanovic nominates @JoshuaKGoldberg for [fixable disable directives](https://github.com/eslint/eslint/pull/14617). :+1: from @btmills and @nzakas.
```
Comment (mdjermanovic) [inline notes/2021/2021-07-01.md:120]:
```suggestion
```

Duplicate, the same line is already above.

File: notes/2021/2021-07-01.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+### Contributor pool
+
+* @nzakas nominates @snitin315. Committer pay will kick in next month in July. :+1: from @btmills and @mdjermanovic.
+* @nzakas nominates @bmish. :+1: from @btmills and @mdjermanovic.
+* @nzakas nominates @jacknapis for Discord help. :+1: from @btmills and @mdjermanovic.
+* @mdjermanovic nominates @brettz9 for 3 ESM PRs. :+1: from @btmills and @nzakas.
+* @mdjermanovic nominates @ota-meshi for work to support the [regex d flag](https://github.com/eslint/eslint/pull/14653). :+1: from @btmills and @nzakas.
+* @mdjermanovic nominates @JoshuaKGoldberg for [fixable disable directives](https://github.com/eslint/eslint/pull/14617). :+1: from @btmills and @nzakas.
+* @mdjermanovic nominates @A-Katopodis for [exit on fatal error](https://github.com/eslint/eslint/pull/14730) :+1: from @btmills and @nzakas.
```
Comment (mdjermanovic) [inline notes/2021/2021-07-01.md:119]:
```suggestion
```

Also duplicate line

Comment (btmills) [reply notes/2021/2021-07-01.md:119]: An extra `.` in vim really got me

**Review by mdjermanovic — APPROVED** [review]

**Review by nzakas — APPROVED** [review]

**Pull request merged**